cmsdb/processes/higgs.py

```diff
@@ -70,52 +90,69 @@
     name="zh",
     id=13100,
     label="ZH",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.9439, {
+            "scale": (0.037j, 0.032j),
+            "pdf": 0.016j,
+        }),  # TODO: only preliminary
+    },
 )
 
 zh_tautau = zh.add_process(
     name="zh_tautau",
     id=13110,
     label=rf"{zh.label}, $H \rightarrow \tau\tau$",
-    xsecs={13: zh.get_xsec(13) * const.br_h.tt},
+    xsecs={ecm: zh.get_xsec(ecm) * const.br_h.tt for ecm in (13, 13.6)},
 )
 
 zh_llbb = zh.add_process(
     name="zh_llbb",
     id=13120,
     label=rf"{zh.label}, $H \rightarrow bb$, $Z \rightarrow ll$",
-    xsecs={13: zh.get_xsec(13) * const.br_h.bb * const.br_z.clep},
+    xsecs={ecm: zh.get_xsec(ecm) * const.br_h.bb * const.br_z.clep for ecm in (13, 13.6)},
 )
 
 zh_qqbb = zh.add_process(
     name="zh_qqbb",
     id=13121,
     label=rf"{zh.label}, $H \rightarrow bb$, $Z \rightarrow qq$",
-    xsecs={13: zh.get_xsec(13) * const.br_h.bb * const.br_z.qq},
+    xsecs={ecm: zh.get_xsec(ecm) * const.br_h.bb * const.br_z.qq for ecm in (13, 13.6)},
 )
 
 wph = vh.add_process(
     name="wph",
     id=13200,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.88696, {
+            "scale": (0.004j, 0.007j),
+            "pdf": 0.018j,
+        }),  # TODO: only preliminary
+    },
 )
 
 wph_tautau = wph.add_process(
     name="wph_tautau",
     id=13210,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={ecm: wph.get_xsec(ecm) * const.br_h.tt for ecm in (13, 13.6)},
 )
 
 wmh = vh.add_process(
     name="wmh",
     id=13300,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.566359, {
+            "scale": (0.004j, 0.007j),
+            "pdf": 0.018j,
```

Comment:
```suggestion
        13.6: Number(0.5677, {  # value for mH=125 GeV
            "scale": (0.004j, 0.007j),
            "pdf": 0.018j,
```

These values were taken from the version 1 of the paper, this was updated in version 2. Additionally, these values are for mH=125.09, I would recommend using 125 GeV for consistency puposes and because the mass used for the generation is probably 125 GeV. I would also add a comment to say for which Higgs mass the value was taken (for bookkeeping)